scss/_base_typography.scss

```diff
@@ -14,6 +14,9 @@
     -webkit-font-smoothing: antialiased;
     // sass-lint:enable no-vendor-prefixes
     color: $color-dark;
+    @if $font-display-option {
+      font-display: $font-display-option;
```

Comment:
Font display needs to be defined in the `@font-face` decoration.

Reply:
moved